src/coreclr/utilcode/util.cpp

```diff
@@ -1321,6 +1321,36 @@ int GetCurrentProcessCpuCount()
                     count = 64;
             }
         }
+
+        JOBOBJECT_CPU_RATE_CONTROL_INFORMATION cpuRateControl;
```

Comment:
Perhaps add some comments here for clarity as to why the JOB object is being queried?

Reply:
There is a comment at the top of this function: "take into account process affinity and CPU quota limit". For Windows we used to ignore the CPU quota limit, which is now enforced right after handling the process affinity, thus fulfilling the existing comment.